Source/AliveDllAO/dllmain.cpp

```diff
@@ -11,24 +11,23 @@ bool RunningAsInjectedDll()
     return true;
 }
 
-extern "C"
+extern "C" {
```

Comment:
this seemed better with the brace on its own line

Reply:
Will fix